client/src/main/java/com/tigrisdata/db/client/Messages.java

```diff
@@ -40,6 +40,7 @@ private Messages() {}
   public static final String UPDATE_FAILED = "Failed to update";
   public static final String DELETE_FAILED = "Failed to delete";
   public static final String READ_FAILED = "Failed to read";
+  public static final String STREAM_FAILED = "Failed to stream";
```

Comment:
Please modify it to `Failed to stream change events`

Reply:
k

Reply:
I'm leaving out "change" since we've said the `/stream` endpoint might be used for more than changes (and hence we didn't use `/changeStream`).